### ISSUE TITLE
Unable to retrieve password from the keychain store.

### ISSUE THREAD

**Issue by bballantyne** [joeferraro/MavensMate-SublimeText#638]
I've read the posts on how to fix this (delete config directory and recreate the MM project), but this doesn't work for me.  I can't save any files to my sandbox.  I'm on Sublime Text 3 build 3083 and upgraded to the latest MM version.  Any ideas?

**Comment by m0gul** [comment]
SAME!

**Comment by SK1724** [comment]
Hi,
I was performing the below steps earlier:
a. Deleted the config file on the existing project.
b. Then right clicked on the existing project and selected Mavensmate > Create New Project
c. After the Mavensmate UI opened, entered the credentials for the same project.
d. After Mavensmate has created the existing project (I should say -- rather updated the existing project :), I tried saving but it was giving the same error.

I closed the Mavensmate app and Sublime.
Then again opened Mavensmate and the existing project.
Wow.. It worked!!

Though I am facing an another issue as the save operation is now pretty slow.. taking 8-9 mins for a simple save.

Regards,
SK
